components/ILIAS/LegalDocuments/tests/LazyProvideTest.php

```diff
@@ -35,9 +36,7 @@ public function testConstruct(): void
         $this->assertInstanceOf(LazyProvide::class, new LazyProvide($this->fail(...)));
     }
 
-    /**
-     * @dataProvider methods
-     */
+    #[Depends('methods')]
```

Comment:
Shouldn't this be `DataProvider`?
```suggestion
    #[DataProvider('methods')]
```